### PR TITLE
SC: add a register and deregister operator test case

### PR DESCRIPTION
**Pull request by jeonghonah**
## Proposed changes

- Add a test case for register and deregister operator

## Types of changes

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

This test case is based on the following document: https://docs.google.com/spreadsheets/d/1aQZVWTNGs6KLHKXUPxHDX5akGUBiGDS9xAexLG6zMfE/edit?pli=1#gid=2142770539

### CODE REVIEW

File: node/sc/multi_bridge_test.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	backend := backends.NewSimulatedBackend(alloc)
+
+	chargeAmount := big.NewInt(10000000)
+	acc.Value = chargeAmount
+	_, tx, b, err := bridge.DeployBridge(acc, backend, false)
+	if err != nil {
+		t.Fatalf("fail to DeployBridge %v", err)
+	}
+	backend.Commit()
+	WaitMined(tx, backend, t)
```
Comment (KimKyungup) [inline node/sc/multi_bridge_test.go]: Please Use CheckWaitMined.

File: node/sc/multi_bridge_test.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	accKey, _ := crypto.GenerateKey()
+	acc := bind.NewKeyedTransactor(accKey)
+
+	alloc := blockchain.GenesisAlloc{acc.From: {Balance: big.NewInt(params.KLAY)}}
+	backend := backends.NewSimulatedBackend(alloc)
+
+	chargeAmount := big.NewInt(10000000)
+	acc.Value = chargeAmount
+	_, tx, b, err := bridge.DeployBridge(acc, backend, false)
+	if err != nil {
```
Comment (KimKyungup) [inline node/sc/multi_bridge_test.go]: How about `assert.noError` ?

File: node/sc/multi_bridge_test.go (hunk 1 above)
Comment (jeonghonah) [reply node/sc/multi_bridge_test.go]: If deploy failed, we should stop here.

File: node/sc/multi_bridge_test.go (hunk 2 above)
Comment (jeonghonah) [reply node/sc/multi_bridge_test.go]: Fixed.

Comment (KimKyungup) [reply node/sc/multi_bridge_test.go]: It seems like that you didn't change `assert.NoError(t, err)`.

**Pull request closed without merging**